### PR TITLE
Enable isort in AutoMM

### PR DESCRIPTION
**Pull request by zhiqiangdon**
*Issue #, if available:*

1. Enable isort in AutoMM.
2. To be consistent with timeseries, black all files in AutoMM.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by yinweisu — APPROVED** [review]
LGTM! Thanks!
Should we also update the README to include instruction on isort?

File: pyproject.toml [hunk 1]
```diff
@@ -17,6 +17,24 @@ known_third_party = [
...
+    "scikit-image",
+    "smart_open",
+    "pytorch_lightning",
+    "torchmetrics",
+    "transformers",
+    "nptyping",
+    "omegaconf",
+    "pytorch-metric-learning",
+    "nlpaug",
+    "nltk",
```
Comment (sxjscience) [inline pyproject.toml:37]: @yinweisu Is it safe to add these new dependencies?

**Review by sxjscience — APPROVED** [review]
LGTM! One question about additional dependencies to pyproject

**Review by bryanyzhu — APPROVED** [review]
LGTM, thanks.

File: pyproject.toml (hunk 1 above)
Comment (yinweisu) [reply pyproject.toml:37]: Not too familiar with it, but seems it's only used by isort. So should be fine

**Comment by FANGAreNotGnu** [comment]
LGTM! Thanks!

**Pull request merged**